src/main/resources/static/index.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0; URL='/search'" />
```

Comment:
This is a small thing, but I believe URL management is very important for an app. Assuming that your app's URL would be something like gascalc.com, you want to make sure that when users go to gascalc.com they are presented with what they expect to see. In this case, we want to automatically redirect people to the search feature as otherwise they are presented with an empty page. In the future, this could be changed to be a landing page potentially.